source/scenes/home/SendAsset/Send/Send.container.tsx

```diff
@@ -62,55 +52,44 @@ import useGasEstimate from 'hooks/useGasEstimate';
 // Header
 ///////////////////////////
 
-
 import Send from './Send';
 
-
 // One billion is the max amount a user is allowed to send.
 const MAX_AMOUNT_NUMBER = 1000000000;
 
 interface IWalletSend {
   initAddress?: string;
-  navigation: any
-};
-
-const SendContainer: FC<IWalletSend> = ({
-  initAddress = '',
-  navigation
-}) => {
+  navigation: any;
+}
 
+const SendContainer: FC<IWalletSend> = ({ initAddress = '', navigation }) => {
   const accountController = getAccountController();
   let isExternalRequest = false;
 
-  if(!!location){
+  if (location) {
```

Comment:
Why was this changed?

Reply:
Prettier made this update. I'll revert it.

Reply:
Sounds good.